src/modules/AdminModule.cpp

```diff
@@ -609,9 +609,20 @@ void AdminModule::handleSetHamMode(const meshtastic_HamParameters &p)
     config.lora.override_duty_cycle = true;
     config.lora.tx_power = p.tx_power;
     config.lora.override_frequency = p.frequency;
+    // Set node info broadcast interval to 10 minutes
+    // For FCC minimum call-sign announcement
+    config.device.node_info_broadcast_secs = 600;
+
+    // Remove PSK of primary channel for plaintext amateur usage
+    auto primaryChannel = channels.getByIndex(channels.getPrimaryIndex());
```

Comment:
![grafik](https://user-images.githubusercontent.com/25002/217681952-87954ae1-6980-4277-9eac-0a8efc61924e.png)

Reply:
If each option is available as a individual config option, encryption (on/default/off), tx power,  frequency,  duty. I don't to understand why a ham mode is required..

Reply:
It shows that we are putting an effort to comply with regional requirements for licensed use.